sycl/test/deprecate_get_count.cpp

```diff
@@ -0,0 +1,12 @@
+// RUN: %clangxx -fsycl -sycl-std=2020 -Xclang -verify -Xclang -verify-ignore-unexpected=note %s -o %t.out
+#include <CL/sycl.hpp>
+
+using namespace cl::sycl;
+
+int main() {
+  // Create a buffer of 4 ints to be used inside the kernel code.
+  buffer<int, 1> Buffer(4);
+  // expected-warning@+1{{'get_count' is deprecated: get_count() is deprecated, please use size() instead}}
```

Comment:
I think it is better to move this check to https://github.com/intel/llvm/blob/sycl/sycl/test/warnings/sycl_2020_deprecations.cpp

Reply:
Moved the check to sycl/test/warnings/sycl_2020_deprecations.cpp and removed this file.